common/src/stack/wizard/gui/boss_config_snack.py

```diff
@@ -58,7 +58,7 @@ def render_network(screen, data):
 		returnExit=0)
 	e3 = Entry(21, text=data.get('Kickstart_PrivateGateway'), scroll=0, \
 		returnExit=0)
-	e4 = Entry(21, text=data.get('Kickstart_PrivateDNSServers'), scroll=0, \
+	e4 = Entry(21, text=data.get('Kickstart_PrivateDNSServers'), scroll=1, \
```

Comment:
what's the effect of the scroll toggle here?

Reply:
It lets the text scroll over so you can type more than 21 characters